tools/failures.py

```diff
@@ -21,7 +21,7 @@ def getRawData(jobtype):
     return cdata['failures']
 
 def getDistinctTuples(jobtype):
-    platforms = ['osx10.6', 'winxp', 'osx10.8', 'win7','linux32','linux64']
+    platforms = ['osx10.6', 'osx-10-6', 'winxp', 'osx10.8', 'osx-10-8', 'win7', 'windows7-32', 'windowsxp', 'win8', 'windows8-64', 'linux32','linux64']
```

Comment:
this is a difference between tbpl and treeherder.  In looking into this I have modified the previous data  so we only have:
osx-10-6
osx-10-8
windowsxp
windows7-32
windows8-64
linux32
linux64

regarding android, we will need to do the same conversion, but I think for now we are good.  The buildtype and testtypes appear to be the same.